repos/spack_repo/builtin/packages/cp2k/package.py

```diff
@@ -118,6 +118,7 @@ class Cp2k(MakefilePackage, CMakePackage, CudaPackage, ROCmPackage):
         " are enabled",
     )
     variant("pytorch", default=False, description="Enable libtorch support")
+    variant("openpmd-api", default=False, description="Enable openPMD support")
```

Comment:
it depends on master so the best way to reflect this is to replace 

`variant("openpmd-api", default=False, description="Enable openPMD support")`

with 

`variant("openpmd-api", default=False, description="Enable openPMD support", when="@2025.3:")`

this addition should only provide the option if and only if the spec specify `cp2k@master`.  The PR can be merged with this addition added.